### PR TITLE
Add underlay for SD data

### PR DESCRIPTION
**Pull request by tjennison-google**
This is almost completely copied (with search/replace) from for the AoU underlay except for the concept_synonym table that isn't part of the SD dataset.

### CODE REVIEW

File: api/src/main/resources/underlays/sd.yaml [hunk 1]
```diff
@@ -0,0 +1,2604 @@
+# An initial OMOP underlay schema for development.
```
Comment (calbach) [inline api/src/main/resources/underlays/sd.yaml:1]: nit: I'm guessing this was copy pasted from the other underlay, if so probably drop

File: api/src/main/resources/underlays/sd.yaml [hunk 2]
```diff
@@ -0,0 +1,2604 @@
...
+        table: observation
+        column: visit_occurrence_id
+  - attribute:
+      entity: observation_occurrence
+      attribute: observation_source_value
+    simpleColumn:
+      columnId:
+        dataset: sd
+        table: observation
+        column: observation_source_value
```
Comment (calbach) [inline api/src/main/resources/underlays/sd.yaml:1978]: Has the schema been downcased now?

File: api/src/main/resources/application.yml [hunk 3]
```diff
@@ -16,4 +16,4 @@ spring:
 
 tanagra:
   underlay:
-    underlay-yaml-files: ["underlays/aou_synthetic.yaml", "underlays/synpuf.yaml"]
+    underlay-yaml-files: ["underlays/aou_synthetic.yaml", "underlays/synpuf.yaml", "underlays/sd.yaml"]
```
Comment (calbach) [inline api/src/main/resources/application.yml:19]: I don't expect our dev instance to be able to load this (I'd say it **shouldn't** be able to), should we be splitting our the configs for local vs dev ?

File: api/src/main/resources/underlays/sd.yaml [hunk 4]
```diff
@@ -0,0 +1,2604 @@
+# An initial OMOP underlay schema for development.
+
+name: sd
+entities:
+  - name: person
```
Comment (calbach) [inline api/src/main/resources/underlays/sd.yaml:5]: As a self-reminder for my next pass on this PR, I want to diff this against the aou synthetic underlay locally

File: workflow/src/main/resources/queries/sd/aa [hunk 5]
```diff
@@ -0,0 +1,23 @@
+-- BuildTextSearchInformation search-strings input query for procedures.
```
Comment (calbach) [inline workflow/src/main/resources/queries/sd/aa:1]: is the filename "aa" a typo?

**Review by calbach — COMMENTED** [review]
Could you please add to this PR description the following:

- Summary of changes to the underlay config, compared to whichever one was your starting point
- Summary of SQL changes, i.e. are these all forks of existing SQL, or are there new queries being added? Were there a lot of small changes needed, or was it mostly just a search/replace?

File: api/src/main/resources/application.yml (hunk 3 above)
Comment (tjennison-google) [reply api/src/main/resources/application.yml:20]: Yes, we will want to split them. I've added a TODO here.

File: workflow/src/main/resources/queries/sd/aa (hunk 5 above)
Comment (tjennison-google) [reply workflow/src/main/resources/queries/sd/aa]: Oops, removed. This was just a temp file.

**Pull request merged**